src/renderers/WebGLRenderer.js

```diff
@@ -1053,6 +1053,37 @@ function WebGLRenderer( parameters = {} ) {
 
 	};
 
+	this.renderShadowMap = function ( lightsWithShadows, scene, camera ) {
```

Comment:
Since the method does not compute the world matrices of `scene` and `camera`, the application has to ensure proper up-to-date matrices, right?

Reply:
True. I've added the auto-update code here. The application can decide whether it runs by setting autoUpdate as usual.